gi-stubs/repository/GdkPixbuf.pyi

```diff
@@ -47,8 +47,10 @@ class Pixbuf(Gio.Icon, Gio.LoadableIcon, GObject.Object):
     def new(*args, **kwargs): ...
     @classmethod
     def new_from_bytes(cls, data: GLib.Bytes, colorspace: Colorspace, has_alpha: bool, bits_per_sample: int, width: int, height: int, rowstride: int) -> Pixbuf: ...
-    def new_from_file(*args, **kwargs): ...
-    def new_from_file_at_scale(*args, **kwargs): ...
+    @classmethod
+    def new_from_file(cls, filename: str) -> Optional[Pixbuf]: ...
+    @classmethod
+    def new_from_file_at_scale(cls, filename: str, width: int, height: int, preserve_aspect_ratio: bool) -> Optional[Pixbuf]: ...
```

Comment:
These two do not return None on fail, because they raise an Exception.

The docs are incorrect here, basically you can ignore the return type everytime it says it raises an exception